### PR TITLE
Load image docs link removed

### PR DESCRIPTION
**Pull request by ashirbad29**
Fixes #1525
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm run test-all`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
               <div class="row step">\
                 <div class="col-md-4 details container-fluid">\
                   <div class="cal collapse in"><p>' +
-                    '<a href="https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#' + step.name + '-module">' + (step.description || '') + '</a>' +
+                    '<a ' + (step.name === "load-image" || 'href="https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#') + step.name + '-module">' + (step.description || '') + '</a>' +
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js:51]: Please use `steps['docs-link']` instead of generating the link.

**Review by HarshKhandeparkar — CHANGES_REQUESTED** [review]

File: examples/lib/defaultHtmlStepUi.js (hunk 1 above)
Comment (ashirbad29) [reply examples/lib/defaultHtmlStepUi.js:51]: it is steps or step?

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js:51]: step. Sorry.

**Comment by ashirbad29** [comment]
how should I proceed next, I have to delete this branch and start a new PR right?

**Comment by HarshKhandeparkar** [comment]
You may edit the branch and it will show up here @ashirbad29

**Comment by ashirbad29** [comment]
@HarshKhandeparkar sorry for so many queries _/\_ 
as I am a complete beginner to open source, can you please help me to edit the branch in the browser and create another PR
I am clueless here :(

**Comment by HarshKhandeparkar** [comment]
Np. Open [ashirbad29/image-sequencer](https://github.com/ashirbad29/image-sequencer/). You can select the branch from the top left corner.
image

Select the branch named `ashirbad29-load-image-docs-link` which is what it is named. Then edit the required file there.

**Comment by ashirbad29** [comment]
@HarshKhandeparkar I committed my changes, please review it

File: examples/lib/defaultHtmlStepUi.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
               <div class="row step">\
                 <div class="col-md-4 details container-fluid">\
                   <div class="cal collapse in"><p>' +
-                    '<a href="https://github.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#' + step.name + '-module">' + (step.description || '') + '</a>' +
+                    '<a ' + (step['docs-link'] ? ('href=' + step['docs-link']) : ('href = "#")) + '>' + (step.description || '') + '</a>' +
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlStepUi.js:51]: Hmm. Looks good. But can we display a `<p>` when `step['docs-link']` does not exist?

Comment (ashirbad29) [reply examples/lib/defaultHtmlStepUi.js:51]: yes, I guess suggest me where should i put that ```<p>``` and what should be the message inside ```<p>```

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js:51]: Oh, I just saw that there is a p already. I guess you can just put `step.description` w/o any `<a>`.  Message inside that p should be `step.description`. Ty.

Comment (ashirbad29) [reply examples/lib/defaultHtmlStepUi.js:51]: you are saying I should conditional render the ```<a>``` right?
if there is no link to the docs then i have to simply display the ```<p>```  with ```step.description```, not the link tag

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlStepUi.js:51]: Yes, precisely.

Comment (ashirbad29) [reply examples/lib/defaultHtmlStepUi.js]: I made the changes you suggested, please review

**Comment by ashirbad29** [comment]
@HarshKhandeparkar I made changes as per your suggestions, please review it

**Review by HarshKhandeparkar — APPROVED** [review]
LGTM. You may use template strings if you wish but this isn't bad :). The rest of it is not using template strings anyway ¯\_(ツ)_/¯

**Comment by ashirbad29** [comment]
yea, it's getting quite a messy, template strings will be clean :)
any suggestion on the code part for me

**Comment by HarshKhandeparkar** [comment]
Nothing, it's perfect.

**Comment by ashirbad29** [comment]
how can i merge this PR?

**Comment by HarshKhandeparkar** [comment]
Well, neither I nor you can merge. We'll have to wait for a *maintainer*.

**Comment by HarshKhandeparkar** [comment]
Wait a sec. You opened the PR in your own fork :joy:

**Comment by ashirbad29** [comment]
well, I was feeling the same from the beginning, i got confused
now how can i start the pr in the original repo

**Pull request merged**